nautobot/extras/models/jobs.py

```diff
@@ -353,7 +264,7 @@ def runnable(self):
 
     @property
     def job_task(self):
-        return app.tasks.get(self.job_class.registered_name, None)
+        return app.tasks[f"{self.module_name}.{self.job_class_name}"]
```

Comment:
do we want the exception to bubble up if the job doesn't exist in the task registry?

Reply:
I've gone back and forth on this. There aren't a lot of places we're calling `.job_task` at present, but most of them will blow up immediately if `job_task` returns None in any case (e.g. `job_model.job_task.apply_async()` in `enqueue_job` so I figured it makes the nature of the failure a bit more intuitive if we do just let it bubble up.